### DISCUSSION TITLE
Pitch Misalignment Query

### DISCUSSION THREAD

**Discussion by PaulKSmith** [OpenFAST/openfast#2142] [Q&A]
Hello,

I am trying to understand which twist I should use to represent pitch misalignment of a single blade? Is it ElastoDyn_blades > "StrcTwst" or AeroDyn15_blade > "BlTwist" ? From my understanding, changing "BlTwist" by constant angle throughout the span will represent the same aerodynamic load as for the pitch misalignment. Please correct me if I am wrong. I am confused what "BlTwist" does then? Does it account for the change in structural properties due to misalignment in overall structural dynamics? I would appreciate if anyone could help me understand this.

**Comment by jjonkman** [comment]
Dear @PaulKSmith,

Regarding the difference between `BlTwist` in AeroDyn and `StrcTwist` in ElastoDyn, see a similar question asked and answered in the following OpenFAST issue: https://github.com/OpenFAST/openfast/issues/2044.

To define a pitch offset between blades in OpenFAST, you could either add an offset to `BlTwist`and `StrcTwist` for a given blade, or you could introduce a pitch offset in the pitch controller.

Best regards,

**Reply by PaulKSmith** [reply]
Hello @jjonkman , Thank you for your kind reply. It took a while for me to go through the refernce.

From #2044 I could understand that "The structural twist in ElastoDyn defines the orientation of the principal axes of bending, which effects the coupling between flapwise and edgewise motions." and "aerodynamic twist in AeroDyn defines the orientation of the chordline, which primarily effects the local angle of attack."
Could you shed some light on how `StrcTwist` in ElastoDyn used to calculate the contribution in flapwise and edgewise motions? Does 1 deg offset in StrcTwist mean 1 deg offset in `BlTwist `?

From #1123 I could understand that aerodynamic forces and structural responses are coupled. Do you think I can use different structural solver to couple with AeroDyn instead of ElastoDyn? I would appreciate if you could provide me some reference. 

Regards,

**Reply by jjonkman** [reply]
Dear @PaulKSmith,

`StrcTwist` in ElastoDyn and `BlTwist` in AeroDyn are specified independently.  So, if you shift `StrctTwist` by 1deg, it will _**not**_ lead to a 1deg shift in the orientation of the chordline.

OpenFAST is modular, and indeed, AeroDyn has already been interfaced to different structural solvers.  Within OpenFAST, AeroDyn can be coupled to ElastoDyn or BeamDyn for the blade structural dynamics.  And apart from OpenFAST, AeroDyn has been coupled to other structural solvers not written by NREL.

Best regards,

**Reply by PaulKSmith** [reply]
Hello @jjonkman , 

Thank you for your reply. So, that means I need to manually give twist to both `StrcTwist` and `BlTwist`. Thanks for the confirmation.

I remember I read in some post that flapwise and edgewise bending are coupled using shape functions ? Is that correct? I could not find that in ElastoDyn manual though. It would be helpful if you could share some insights on this.

Regards,

**Reply by jjonkman** [reply]
Dear @PaulKSmith,

Yes, I agree that ElastoDyn will couple the flapwise and edgewise motions based on the structural pretwist.  See the following topic on our forum for more information: https://forums.nrel.gov/t/coupled-blade-modes-in-fast/314.

Best regards,

**Reply by PaulKSmith** [reply]
Hello @jjonkman 

Thanks for sharing the post link. It helped clearing doubts. 
I have another doubt and could not find any reference in any other posts. I would be grateful if you could help. I am trying to calculate loads on blade individual nodes using `B1N1Fx` and 
`B1N1Fy`. The outputs are in `N/m`. Does it mean I need to multiply effective length to get in `N` at a specific node? Also, can I use effective length as 50% of the node-to-node distance on both side ?


Regards,

**Reply by jjonkman** [reply]
Dear @PaulKSmith,

Indeed, AeroDyn's aerodynamic force outputs are expressed as distributed forces per unit length (in N/m).  If you want to lump the loads as point loads at nodes (in N), you could multiply by an appropriate length, e.g, half of the distance to the previous inboard node plus half the distance to the next outboard node.

Best regards,

**Reply by PaulKSmith** [reply]
Hello @jjonkman ,

Thank you so much for your time and effort. 

Regards,
